lib/parse.js

```diff
@@ -2236,17 +2236,86 @@ function parse($TEXT, options) {
         })
     }
 
+    function import_nameAsterisk() {
+        var start = S.token;
+        var foreign_name;
+        var name;
+
+        next();
+
+        var end = prev();
+
+        name = new AST_Symbol({
```

Comment:
In this case it is used for the asterisk in `export * from "a.js"`, which is a wildcard alternative for `export {A, B, C} from "a.js"`. I have replaced it with more appropriate `AST_SymbolImport`, but have to admit I am not sure if this should be a symbol at all.